### PR TITLE
#108 | View event rating

### PR DESCRIPTION
**Pull request by AhmedSoliman7**
1) added link to the organization's profile hosting the event.
2) added rate for the organization's event view if the event ended and there is a rate .

### CODE REVIEW

File: app/Http/Controllers/Auth/OrganizationAuthController.php [hunk 1]
```diff
@@ -15,7 +15,7 @@
 use Elasticsearch\ClientBuilder as elasticClientBuilder;
 
 use Auth;
-use Input;
```
Comment (AhmadElsagheer) [inline app/Http/Controllers/Auth/OrganizationAuthController.php:18]: restore

File: resources/views/event/review/create.blade.php [hunk 2]
```diff
@@ -1,9 +1,32 @@
 @extends('layouts.app')
 
-@section('content')
-    <h1> create organization review on {{ $event->name }}</h1>
-    {!! Form::open(['url' => 'event/'.$event->id.'/review']) !!}
-        @include('event.review.partials.form', ['submitButtonText' => 'Add Review!'])
-    {!! Form::close() !!}
-    
+        @section('content')
```
Comment (AhmadElsagheer) [inline resources/views/event/review/create.blade.php]: ??

File: app/Http/Services/EventReviewService.php [hunk 3]
```diff
@@ -7,6 +7,8 @@
 use App\Http\Requests\EventReviewRequest;
 
 use App\Event;
+use App\Challenge;
```
Comment (AhmadElsagheer) [inline app/Http/Services/EventReviewService.php]: remove these lines

File: app/Http/Controllers/OrganizationController.php [hunk 4]
```diff
@@ -0,0 +1,150 @@
+<?php
```
Comment (AhmadElsagheer) [inline app/Http/Controllers/OrganizationController.php]: remove this file

File: app/Http/Controllers/Event/EventController.php [hunk 5]
```diff
@@ -57,7 +57,9 @@ public function index($organization_id)
 	 */
 	public function show($id)
 	{
-        $event = Event::findOrFail($id);
+    		$event = Event::findOrFail($id);
+		$event->organizationName = $event->organization()->name;
+		$event->organization_id = $event->organization()->id;
 		$creator = null;
```
Comment (AhmadElsagheer) [inline app/Http/Controllers/Event/EventController.php:61]: you can retrieve them directly in the view. No need to add more attributes to $event

**Comment by AhmadElsagheer** [comment]
Check **Files changed**. There are to many unnecessary files that have been already removed.

File: resources/views/event/review/create.blade.php (hunk 2 above)
Comment (AhmedSoliman7) [inline resources/views/event/review/create.blade.php]: "??" ? Give me comment .
 when I pulled I found overwriting this form. What should i do.

File: app/Http/Services/EventReviewService.php (hunk 3 above)
Comment (AhmedSoliman7) [inline app/Http/Services/EventReviewService.php]: You mean all these three lines ?

File: app/Http/Controllers/OrganizationController.php (hunk 4 above)
Comment (AhmedSoliman7) [inline app/Http/Controllers/OrganizationController.php]: I removed it , but it gives errors when you access any organization's profile.

File: resources/views/event/review/create.blade.php (hunk 2 above)
Comment (AhmadElsagheer) [inline resources/views/event/review/create.blade.php]: You should always check the changed files. If you didn't make these changes, then **don't leave them and restore everything as it was**

File: resources/views/event/review/index.blade.php [hunk 6]
```diff
@@ -20,7 +20,6 @@
             Rate: {{$review->rating}}
             @if (Auth::user() && Auth::user()->eventReviews()->find($review->id))
                 <a href="/event/{{$event->id}}/review/{{$review->id}}/edit">edit</a>
-            .
```
Comment (AhmadElsagheer) [inline resources/views/event/review/index.blade.php:23]: restore this dot

File: app/Http/Controllers/OrganizationController.php (hunk 4 above)
Comment (AhmadElsagheer) [inline app/Http/Controllers/OrganizationController.php]: The file is not removed. It shouldn't give you an error

**Pull request merged**